src/module_1/module_1_meteo_api.py

```diff
@@ -1,7 +1,104 @@
-""" This is a dummy example """
+import requests
+import json
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+from datetime import timedelta
+from ratelimit import limits, sleep_and_retry
+
+
+COORDINATES = {
+    "Madrid": {"latitude": 40.416775, "longitude": -3.703790},
+    "London": {"latitude": 51.507351, "longitude": -0.127758},
+    "Rio": {"latitude": -22.906847, "longitude": -43.172896},
+}
+
+VARIABLES = "temperature_2m_mean,precipitation_sum,soil_moisture_0_to_10cm_mean"
```

Comment:
Even if you are gonna use this as an string later, use a list here. A raw string is error prone, hard to read and hard to modify